edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/validation/ValidUtilizedTransportEquipment.java

```diff
@@ -0,0 +1,37 @@
+package org.dcsa.edocumentation.transferobjects.validation;
+
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Validate that a given String field contains a valid vessel IMO number
```

Comment:
Looks like copy-waste given it does not validate IMO numbers.

Reply:
fixed